[Doc]Add statement for supporting remtoe redis (#16869)
ray-project/ray: doc/source/cluster/cloud.rst

The command will print out the address of the Redis server that was started
(the local node IP address plus the port number you specified).

.. note::

>     If you already has remote redis instances, you can specify `--address=ip1:port1,ip2:port2...` to use them. The first one is primary and left are shards. Ray will use them if the primary one if it is reachalbe, otherwise fallback to create on local.

Reviewer: ```suggestion
    If you already has remote redis instances, you can specify `--address=ip1:port1,ip2:port2...` to use them. The first one is primary and rest are shards. Ray will create a redis instance if the default is unreachable.
```
